test/tests/deformation/mesh-velocity-areas/tests

```diff
@@ -0,0 +1,17 @@
+[Tests]
+  [mesh_velocity_areas]
+    type = CSVDiff
+    input = pipe.i
+    csvdiff = 'pipe_out_nek0.csv pipe_out.csv'
+    min_parallel = 4
+    abs_zero = 1e-6
+    rel_err = 5e-4
```

Comment:
If this is a new test, why make the relative error kind of loose? I'd advocate for removing line 8 so that we have a stronger test, unless there's a reason otherwise?